client/site-profiler/components/get-report-form/styles.scss

```diff
@@ -34,6 +33,12 @@
 	margin: 0 auto;
 	max-width: 1224px;
 
+	@media (max-width: $break-mobile) {
+		height: 100vh;
+		height: 100svh;
```

Comment:
This doesn't seem working as expected in ios 

![CleanShot 2024-06-04 at 19 10 26@2x](https://github.com/Automattic/wp-calypso/assets/12430020/2d89996c-dbbf-47e8-97a8-fb8c5bf02d61)

Have you tried using `dvh` in this container? Also, is there a reason for `height: 100vh;` fallback? 

https://www.sabhya.dev/handling-ios-safari-toolbar-for-full-height-web-content

Reply:
> This doesn't seem working as expected in ios

Can you explain how have you reproduced it? It works on my phone (ios) using [this link](https://calypso.live/?image=registry.a8c.com/calypso/app:build-109905). I have tested also using Developer Tools as you can see in the video emulating IPhone SE.

>Have you tried using dvh in this container?

Yes, but `dvh` can cause some bouncing when scrolling, so that is why I have opted to use `svh`. Here there is a nice video explaining the differences https://www.youtube.com/watch?v=ru3U8MHbFFI

>Also, is there a reason for height: 100vh; fallback?

The reason for the fallback is to cover the browsers that do not support `svh` or `dvh`, although it is covered by 92% so I can remove it (https://caniuse.com/mdn-css_types_length_viewport_percentage_units_small)

Reply:
I used the xcode simulator but I can see the same on my phone. The problem starts when you scroll down and then click to open the form. I think `dvh` would work better in IOS due to the url bar getting minimised.

Reply:
I have changed to `dvh` as suggested, but I can observe that after scrolling, the screen does not take the full height of the screen. The `Submit` button is reachable though as the form can be scrolled. 

I will continue investigating this and I will try to setup the `xcode` simulator in order to debug this as currently I am using the Calypso live generated links.

Reply:
I have fixed the issue by removing the top margin and increasing the `z-index` so the form takes the full height. This is a video with the simulator in ios

https://github.com/Automattic/wp-calypso/assets/3519124/9995326a-4ddf-452f-b1b8-604887a8157e

@cpapazoglou , please let me know if you identify any further issues

Reply:
It works great now 👏 👏 👏